src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -535,7 +538,8 @@ Namespace Microsoft.CodeAnalysis.Semantics
         Private Function CreateBoundUserDefinedConversionOperation(boundUserDefinedConversion As BoundUserDefinedConversion) As IConversionExpression
             Dim operand As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundUserDefinedConversion.Operand))
             Dim syntax As SyntaxNode = boundUserDefinedConversion.Syntax
-            Dim conversion As Conversion = _semanticModel.GetConversion(syntax)
+            Dim summary As BoundNodeSummary = New BoundNodeSummary(boundUserDefinedConversion.Operand, boundUserDefinedConversion, Nothing)
+            Dim conversion As Conversion = _semanticModel.GetTypeInfoForNode(summary).ImplicitConversion
```

Comment:
>Dim conversion As Conversion = _semanticModel.GetTypeInfoForNode(summary).ImplicitConversion [](start = 12, length = 92)

Same issue. Also, it feels suspicious that we create a dedicated IOperation node for a BoundUserDefinedConversion. I believe it is always paired with BoundConversion node and we should be creating a single IOperation per pair.